### DISCUSSION TITLE
[Bug]: build ANDROID failed " openssl/asn1.h' file not found"

### DISCUSSION THREAD

**Discussion by c14096073** [alibaba/xquic#288] [General]
### What happened?

使用sh xqc_build.sh ios/android <build_dir> <artifact_dir> <ssl_path> 編譯時發生錯誤
 " openssl/asn1.h' file not found"，嘗試過" sudo apt-get install libssl-dev"依然沒有用，想請問該如何解決這個問題，謝謝

### Steps To Reproduce

![螢幕擷取畫面 (2)](https://user-images.githubusercontent.com/115781883/220501517-e6592a43-a38f-453f-963a-7f980d072d8a.png)

**Comment by ruiqizhou** [comment]
在编译 BoringSSL 时报错，建议降低 BoringSSL版本或者查阅 BoringSSL 官方文档
